### ISSUE TITLE
Camera.bounds() causes game object's position to increase enormously

### ISSUE THREAD

**Issue by ghost** [britzl/defold-orthographic#31]
My window resolution is set to 480 x 320. I’m using the FIXED_AUTO projection.

If I use the function:
camera.bounds(hash("/camera"), 0, 320, 480, 0)

The camera game object’s position skyrockets into the void to like <3e23, 15e47, 0> and continues rising like this. I’d like to add a feature where you can push a button and the camera “zooms out” and perfectly displays the entire level.

Here is a snippet of the collection hierarchy:
https://i.imgur.com/eUOv04p.png

Here is a snippet of the camera's properties:
https://i.imgur.com/llXsTcu.png

All properties omitted are set to 0 (bounds, deadzone, etc).
When I use the camera.bounds() function, the camera is not following any game object, nor is the camera a child of any game object. If I use camera.follow(player) then use camera.bounds(), the same issue occurs.

**Comment by britzl** [comment]
I'd like to understand how this happens. I have added a new basic example which sets up camera bounds and it works as expected. Could you please try this example and make the required changes to it to get the behaviour you are seeing?

https://github.com/britzl/defold-orthographic/tree/master/example/bounds

**Comment by ghost** [comment]
No idea what's going on. Looks like your setup is the same as mine. Maybe it has something to do with resizing the window..?

Actually, I thought bounds() was supposed to display a specified area of the map, but I guess I was wrong. It just limits the position of the camera within the bounds. But this still doesn't make much sense. My camera is located at position <240, 160>. Since the bounds are set to <0, 480, 320, 0>, there shouldn't be an issue.

Here's my code for setting the bounds, nice and simple:

elseif action_id == hash("key_e") and action.pressed then
		camera.unfollow(hash("/camera"))
		go.set_position(vmath.vector3(240, 160, 0), "camera")
		camera.bounds(hash("/camera"), 0, 320, 480, 0)

(Sorry, don't know how to format code in Git.)

If you'd prefer, I can send you a copy of my project.

**Comment by britzl** [comment]
Please share your project with me (bjorn@defold.se)

**Comment by SamsTheGreatest** [comment]
@britzl What was the issue? I am also trying to deal with bounds. I have a tile map with 60x40 cells, each 16x16 pixels. If I set 
```
top,right = #self.matrix*tile_size,#self.matrix[1]*tile_size
camera.bounds(self.camera_id, 0, top, right, 0)
```
with `tile_size=4` the bounds are set correctly. If I set `tile_size=16` bounds are too far out. `self.matrix` is a matrix of size `40x60`.

Could it be an issue that something shall be converted from `world` to `screen`? Bounds are defined on-screen or in the world?
